topics/control-flow/examples/rocket_launch2.cpp

```diff
@@ -1,42 +1,41 @@
+
 // ======================
 // = Main - Entry Point =
 // ======================
 
 // Run the rocket launch...
 int main()
 {
-    //declare variables
+    // declare variables
     int rocket_x, rocket_y;
     bool thrusters_on = false;
     color rocket_color;
-    
+
     open_audio();
-    open_graphics_window("Rocket Launch", 800, 600);
-    load_default_colors();
-    
-    //initialise the rocket position, color
-    rocket_color = ColorBlue;
+    open_window("Rocket Launch", 800, 600);
+
+    // initialise the rocket position, color
+    rocket_color = color_blue();
     rocket_y = screen_height() - ROCKET_HEIGHT;
     rocket_x = (screen_width() - ROCKET_WIDTH) / 2;
-    
+
     load_resources();
-    
-    while(!window_close_requested() && !key_typed(VK_ESCAPE))
+
+    while (!window_close_requested("Rocket Launch") && !key_typed(ESCAPE_KEY))
     {
-        //let swingame process user input
+        // let splashkit process user input
         process_events();
-        
-        //Update the rocket
-        update_rocket(rocket_y, thrusters_on);  // passed by reference (C++)
-        
-        //draw the rocket on the screen!
+
+        // Update the rocket
+        update_rocket(rocket_y, thrusters_on); // parameters passed by reference (C++)
+
+        // draw the rocket on the screen!
         clear_screen();
         draw_rocket(rocket_color, rocket_x, rocket_y, thrusters_on);
         refresh_screen();
     }
-    
+
     close_audio();
-    
-    release_all_resources();
+
     return 0;
-}
+}
```

Comment:
**suggestion:** Could we please ensure that there is an EOL for all files in this changeset? 🙏🏼

Reply:
There it is: [refactor: add EOL to everyline](https://github.com/thoth-tech/programming-arcana/pull/1/commits/ac15f662930ada1330ceaa62b51a3648f15caa88). I am new to c and c++ so I did not know this could be an issue, did a bit research and figured it out. Thanks for let me know.